packages/discordx/examples/button/commands/game.ts

```diff
@@ -66,9 +65,8 @@ class spcProposition {
 const defaultChoice = new spcProposition(spcChoice.Stone, "💎", "spc-stone");
 
 @Discord()
-@Bot("alexa")
 export abstract class StonePaperScissor {
-  @Slash("stonepaperscissor", {
+  @Slash("rock-paper-scissors", {
     description:
       "What could be more fun than play Rock Paper Scissor with a bot?",
```

Comment:
```suggestion
      "What could be more fun than play Rock Paper Scissors with a bot?",
```

Reply:
had to change more then this, you can review this again.